src/main/steps/c100-rebuild/safety-concerns/other-concerns/content.ts

```diff
@@ -0,0 +1,87 @@
+import { YesOrNo } from '../../../../app/case/definition';
+import { PageContent } from '../../../../app/controller/GetController';
+import { FormContent } from '../../../../app/form/Form';
+import { isFieldFilledIn, isTextAreaValid } from '../../../../app/form/validation';
+import { CommonContent } from '../../../common/common.content';
+
+const en = {
+  section: 'Safety concerns',
+  title: 'Do you have any other concerns about the children’s safety and wellbeing?',
+  hint: 'For example, their basic needs are not being met (known as child neglect) or you’re worried about someone they may have contact with.',
+  one: 'Yes',
+  two: 'No',
+  summaryText: 'Contacts for help',
+  errors: {
+    childSafetyConcerns: {
+      required: 'Please select an answer',
+    },
+    childSafetyConcernsDetails: {
+      required: 'Describe what concerns you have about the children’s safety and wellbeing',
+    },
+  },
+};
+
+const cy: typeof en = {
+  section: 'Safety concerns',
+  title: 'Do you have any other concerns about the children’s safety and wellbeing?',
+  hint: 'For example, their basic needs are not being met (known as child neglect) or you’re worried about someone they may have contact with.',
+  one: 'Yes',
+  two: 'No',
+  summaryText: 'Contacts for help',
+  errors: {
+    childSafetyConcerns: {
+      required: 'Please select an answer',
+    },
+    childSafetyConcernsDetails: {
+      required: 'Describe what concerns you have about the children’s safety and wellbeing',
+    },
+  },
+};
+
+const languages = {
+  en,
+  cy,
+};
+
+export const form: FormContent = {
+  fields: {
+    childSafetyConcerns: {
```

Comment:
@ShakiraKP Can we please follow the naming conventions defined for each features ?
Please prefix these fields with c1A_